src/main/java/com/codeborne/selenide/commands/SetValue.java

```diff
@@ -26,6 +27,12 @@ public SetValue(SelectOptionByValue selectOptionByValue, SelectRadio selectRadio
   @Override
   public WebElement execute(SelenideElement proxy, WebElementSource locator, Object[] args) {
     String text = (String) args[0];
+
+    if (args[0] instanceof HiddenString) {
```

Comment:
Modifying default setValue method you will push user to pass HiddenString instead just string

Reply:
SetValue.execute method accepts Object[] args.

Where the first argument can be String (backwards compatibility) or HiddenString (added feature).

In SelenideElement there are existing setValue and new setValue methods:
SelenideElement setValue(String text);
SelenideElement setValue(HiddenString hiddenString);
Both of these methods are supported by SetValue.execute(...).